app/javascript/components/App.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import Greeting from "./greeting"
+
+function App() {
+  return (
+    <div>
+      <Greeting />
+    </div>
+    );
+}
+
+export default App;
```

Comment:
- [ ] Great job so far, however, It will be great for the `App` component to have a route that leads to the `Greeting` component. This is a major requirement for this project to be approved. ⬇️ and

![Screenshot from 2023-01-03 03-31-44](https://user-images.githubusercontent.com/79058364/210289974-c72f4748-1371-4647-93eb-885b397f96af.png)